core/src/impl/Kokkos_Profiling.cpp

```diff
@@ -76,10 +78,19 @@ static stopProfileSectionFunction stopSectionCallee       = nullptr;
 static destroyProfileSectionFunction destroySectionCallee = nullptr;
 
 static profileEventFunction profileEventCallee = nullptr;
+} // end namespace Profiling
 
-SpaceHandle::SpaceHandle(const char* space_name) {
-  strncpy(name, space_name, 64);
-}
+namespace Tuning {
+
+static tuningVariableDeclarationFunction tuningVariableDeclarationCallee = nullptr;
+static tuningVariableValueFunction tuningVariableValueCallee = nullptr;
+static contextVariableDeclarationFunction contextVariableDeclarationCallee = nullptr;
+static contextVariableValueFunction contextVariableValueCallee = nullptr;
+static contextEndFunction contextEndCallee = nullptr;
```

Comment:
Following the old conventions. If nobody objects to your changes, I'll implement them

Reply:
I'm not going to make the thread_local change yet, it's a part of the interface that isn't breaking anything, and it would be a change from what it used to be